analysis/tc_book-server_verified.go

```diff
@@ -0,0 +1,76 @@
+package analysis
+
+import (
+	"github.com/konveyor/go-konveyor-tests/hack/addon"
+	"github.com/konveyor/tackle2-hub/api"
+)
+
+var BookServerApp = api.Application{
```

Comment:
The application variable exists here: https://github.com/konveyor/go-konveyor-tests/blob/main/data/application_data.go